pyproject.toml

```diff
@@ -39,6 +35,7 @@ streamlit = {version = "^1.29.0", optional = true}
 fastapi-sso = { version = "^0.10.0", optional = true }
 PyJWT = { version = "^2.8.0", optional = true }
 python-multipart = { version = "^0.0.6", optional = true }
+google-cloud-aiplatform = "^1.43.0"
```

Comment:
hi @GuillermoBlasco, does this need to be in the client sdk's pyproject.toml? the issue you faced seemed docker env specific

Reply:
it is in the proxy. Read in the PR Comment:
<img width="919" alt="image" src="https://github.com/BerriAI/litellm/assets/1889971/01e4928d-7068-4b2f-a68c-41b89e2b2ad3">

Reply:
Do this `google-cloud-aiplatform = { version = "^1.43.0", optional = true }`

I believe that should fix it